docs/content/connectors/formats/json.md

```diff
@@ -0,0 +1,27 @@
+# JSON Format
+
+The JSON format allows to read and write JSON data based on an JSON schema. The JSON 
+schema is derived from table schema.
+
+## Format Options
+
+| key                | Required | default | type    | Description                                                                                          |
+|--------------------|----------|---------|---------|------------------------------------------------------------------------------------------------------|
+| ignore_parse_error | Optional | True    | boolean | Skip fields and rows with parse errors instead of failing. Fields are set to null in case of errors. |
+
+## Data Type Mapping
+
+The following table lists the type mapping from Flink type to JSON type.
+
+| Feathub type | JSON type                     |
+|--------------|-------------------------------|
+| BYTES        | string with encoding: base64  |
+| STRING       | string                        |
+| INT32        | number                        |
```

Comment:
It seems inconsistent to use int32, int64, float and double.

How about we rename INT32 as INT and rename `INT64` as Long, for consistency with Spark DataType [1]?

Flink uses `BIGINT` to denote "Data type of an 8-byte signed integer". However, Java uses BigInteger to represent arbitrary-precision integers [2]. Thus it seems better to use long.

[1] https://spark.apache.org/docs/latest/sql-ref-datatypes.html
[2] https://www.baeldung.com/java-biginteger

Reply:
I find out that the type names in the docs were inaccurate.  The naming of the Feathub type is consistent, int32, int64, float32, and float64.

I updated the docs so they are consistent now.